great_expectations_cloud/agent/models.py

```diff
@@ -38,6 +38,7 @@ class RunMissingnessDataAssistantEvent(RunDataAssistantEvent):
 
 class RunCheckpointEvent(EventBase):
     type: Literal["run_checkpoint_request"] = "run_checkpoint_request"
+    datasource_names_to_asset_names: dict
```

Comment:
```suggestion
    datasource_names_to_asset_names: dict[str, str]
```

Reply:
I think this syntax only works with Python 3.9 and up. I had tried it before (and forgot and just tried it again with the suggestion above :) and the Python 3.8 tests fail with `TypeError: 'type' object is not subscriptable`

Not sure if there is another way around this.

Reply:
```python
from typing import Dict, Set

...

datasource_names_to_asset_names: Dict[str, Set[str]]
```